script/test/cmd/cmd_test.go

```diff
@@ -0,0 +1,775 @@
+	cmdOut, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Println("")
+		return "", errors.Wrap(err, "Error getting branch name")
+	}
+	branch = string(cmdOut)
+	return branch, nil
+}
+
+//UpdateUri Replaces variables with current branch and uri
+func UpdateURI(uri, branch, inputfile, output string) error {
+	input, _ := ioutil.ReadFile(inputfile)
+	file := string(input)
+	r, _ := regexp.Compile("%URI%")
+	f := r.ReplaceAllString(file, uri)
+	k, _ := regexp.Compile("%REF%")
+	g := k.ReplaceAllString(f, branch)
+	err := ioutil.WriteFile(output, []byte(g), 0644)
+	if err != nil {
+		return errors.Wrapf(err, "Error updating the file with %s and %s", uri, branch)
+	}
+	return nil
+}
+
+//Get origin
+func GitOrigin() (origin, branch string, error error) {
+	cmdName := "git"
+	cmdArgs := []string{"remote", "get-url", "origin"}
+	cmdOut, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Println("")
+		return "", "", errors.Wrap(err, "Error getting origin.")
+	}
+	origin = string(cmdOut)
+	origin = strings.TrimSpace(origin)
+	if !strings.Contains(origin, ".git") {
+		origin = origin + ".git"
+	}
+	brch, err := Git()
+	if err != nil {
+		log.Printf("%+v", err)
+		return
+	}
+	branch = strings.TrimSpace(brch)
+	fmt.Println("Test regarding build context running kompose from various directories")
+	fmt.Println("Buildconfig using " + origin + "::" + branch + " as source.")
+	return origin, branch, nil
+}
+
+// Kompose command for buildContext
+func KomposeCommand(command, provider, outFile, dockerComposeFile string) (error error) {
+	file := strings.Split(dockerComposeFile, "/")
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).Output()
+	if err != nil {
+		return errors.Wrapf(err, "Error in %s application", file[1])
+	}
+	er := ioutil.WriteFile(outFile, out, 0644)
+	if er != nil {
+		return errors.Wrapf(err, "Error in writing file.")
+	}
+	return nil
+}
+
+// Kompose command
+func Kompose(command, provider, dockerComposeFile, result string) (cmd, output string, error error) {
+	b, error := ioutil.ReadFile(result)
+	if error != nil {
+		return "", "", errors.Wrap(error, "Error reading file")
+	}
+	file := strings.Split(dockerComposeFile, "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Println("\x1b[31;1m===> Test Fail <===\x1b[0m")
+		fmt.Println("Test Fail for", file[1])
+		return "", "", errors.Wrapf(err, "Error in %s application", file[1])
+	}
+	cmd = string(out)
+	fmt.Println("")
+	return cmd, output, nil
+}
+
+// Kompose command for Multiple files
+func MultipleFiles(command, provider, dockerComposeFileOne, dockerComposeFiletwo, result string) (cmd, output string, error error) {
```

Comment:
`[]string` will not work here as we provide multiple compose files using `f <file1> -f <file2>`